tests/_internal/test_client.py

```diff
@@ -208,6 +208,76 @@ async def test_async_client_list_clients_public():
     _assert_pagination_api_calls(route)
 
 
+@respx.mock
+def test_client_list_links_internal():
+    # Given
+    route = respx.get("http://test.example.com:1234/v1/links")
+    route.side_effect = LINKS_RESPONSES
+
+    cl = client.Client("http://test.example.com:1234")
+
+    # When
+    links = cl.list_links()
+
+    # Then
+
+    assert links == EXPECTED_LINKS_RECORDS
+    _assert_pagination_api_calls(route)
+
+
+@respx.mock
+def test_client_list_links_public():
+    # Given
+    route = respx.get("https://test.example.com/.ai.h2o.cloud.discovery/v1/links")
+    route.side_effect = LINKS_RESPONSES
+
+    cl = client.Client("https://test.example.com/.ai.h2o.cloud.discovery")
+
+    # When
+    links = cl.list_links()
+
+    # Then
+
+    assert links == EXPECTED_LINKS_RECORDS
+    _assert_pagination_api_calls(route)
+
+
+@respx.mock
+@pytest.mark.asyncio
+async def test_async_client_list_links_internal():
+    # Given
+    route = respx.get("https://test.example.com:1234/v1/links")
+    route.side_effect = LINKS_RESPONSES
+
+    cl = client.AsyncClient("https://test.example.com:1234")
+
+    # When
+    links = await cl.list_links()
+
+    # Then
+
+    assert links == EXPECTED_LINKS_RECORDS
```

Comment:
```suggestion
    # Then
    assert links == EXPECTED_LINKS_RECORDS
```
Just in case newline not desired.